README.md

```diff
@@ -1,5 +1,19 @@
 # Virtual Schema Installer
 
+[![Build Status](https://github.com/exasol/virtual-schema-installer/actions/workflows/ci-build.yml/badge.svg)](https://github.com/exasol/virtual-schema-installer/actions/workflows/ci-build.yml)
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
+
+[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=security_rating)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
+[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
+[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
+[![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=sqale_index)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
+
+[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=code_smells)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=coverage)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
+[![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=duplicated_lines_density)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
+[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=ncloc)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)'
```

Comment:
```suggestion
[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Avirtual-schema-installer&metric=ncloc)](https://sonarcloud.io/dashboard?id=com.exasol%3Avirtual-schema-installer)
```

Reply:
Maybe I should change the output somehow. Having this large block in quotes is confusing. But did not have a better idea yet...

Reply:
Maybe just adding a couple of empty lines at the start and the end of the block could help?